torchvision/datasets/celeba.py

```diff
@@ -38,6 +38,12 @@ class CelebA(VisionDataset):
         download (bool, optional): If true, downloads the dataset from the internet and
             puts it in root directory. If dataset is already downloaded, it is not
             downloaded again.
+
+            .. warning::
```

Comment:
Shouldn't we replace the original description with the warning? With this path the original description is wrong, since we will error out earlier.